packages/did/src/did.ts

```diff
@@ -36,6 +39,39 @@ export const decodeFromDid = (tangleid: Did): MnidModel => {
   return mamParsed;
 };
 
+/**
+ * Checks if given URL is a DID URL.
+ * @function isDidUrl
+ * @param {string} url - the URL to be checked.
+ * @return {boolean} The boolean that represent the url whether DID URL.
+ */
+export const isDidUrl = (url: string): boolean => {
+  return DID_URL_REGEX.test(url);
```

Comment:
Should we check from a known list of existing methods?

Reply:
Check the method whether exist or not would be handled by another function, but we don't have the use case for that function for now.